src/pymovements/utils/parsing.py

```diff
@@ -209,4 +247,50 @@ def parse_eyelink(
         schema_overrides=schema_overrides,
     )
 
-    return df
+    return df, metadata
```

Comment:
Please cast `metadata: defaultdict` to a simple `dict` when returning.

A defaultdict can result in unexpected behavior and the documentation says it returns a plain dictionary.